inspirehep/modules/references/processors.py

```diff
@@ -81,16 +81,26 @@ def _split_refextract_authors_str(authors_str):
         lambda a: a.startswith(','),
         lambda a: len(a) == 1
     ]
-    res = [r for r in res if all(not f(r) for f in filters)]
+    res = [_improve_author(r) for r in res if all(not f(r) for f in filters)]
 
     return res
 
 
+def _improve_author(author_names):
+    res = author_names.split(' ')[-1] + ', '
+    for name in author_names.split(' ')[0:-1]:
+        res += name
+    return res
```

Comment:
This logic is obviously too naive to really handle names. Proper treating of this problem is deferred to the time in which we seriously integrate `nameparser` as a util in INSPIRE.